src/index.js

```diff
@@ -20,12 +20,30 @@ const ModalApp = ({ options:{ useModal, isModalOpen, onModalRequestClose, ...oth
     <Router options={otherOptions} {...otherProps}/>
   </Modal>
 
-const Container = ({ options }) =>
-  <ReduxProvider store={store}>
-    <LocaleProvider language={options.language}>
-      <ModalApp options={options} />
-    </LocaleProvider>
-  </ReduxProvider>
+class Container extends Component {
+  componentDidMount() {
+    this.prepareInitialStore()
+  }
+
+  prepareInitialStore = () => {
```

Comment:
This logic would be a lot cleaner out in a separate `boot` directory or something, but imo it seems overkill when there's only 1 thing to do on load atm

But something to refactor in the future, if more stuff gets added to this, maybe?

Reply:
So only the `smsNumber` is in the store on initialise? Where do we store the rest of the config?

Reply:
The rest of the config is just passed around as an options object

But the `smsNumber` is used in the store (the user can edit it, and that updated value becomes the new store value)

So basically, `smsNumber` is already a value in the store. We just want to set its _initial value_. None of the other options have anything to do with values in the store